test/unit/logic/transaction.js

```diff
@@ -751,7 +751,7 @@ describe('transaction', () => {
 			});
 		});
 
-		it('should return error when transaction amount is invalid', done => {
+		it('should not return error when transaction amount is bigger than total amount of Lisk tokens', done => {
```

Comment:
This is misleading since the test expects that there is an error.

Reply:
@willclarktech improved it.